### PR TITLE
jormungandr 0.8.0-rc1 - 0.8.0-rc4

### PR DESCRIPTION
**Pull request by piotr-iohk**
# Issue Number

# Overview

- [ ] I have updated `jormungandr.nix` -> jormungandr 0.8.0-rc1 - 0.8.0-rc4
- [ ] I have regenrated arbitrary-blocks for gloden tests

# Comments

### CODE REVIEW

**Comment by piotr-iohk** [comment]
bors try

**Review by Anviking — APPROVED** [review]

**Comment by KtorZ** [comment]
_working on the breaking changes_

**Comment by piotr-iohk** [comment]
In the meantime I changed timeout on `eventually` function in DSL to be 1 min (instead of 10 mins). 10 mins is definitely too long...

**Comment by KtorZ** [comment]
@piotr-iohk but one minute is unfortunately too short in a few cases (like the stake pool list scenarios).

**Comment by piotr-iohk** [comment]
@KtorZ, I changed to 3 minutes. I think it should be enough.

File: lib/core-integration/src/Test/Integration/Framework/DSL.hs [hunk 1]
```diff
@@ -875,10 +875,10 @@ eventuallyUsingDelay
     -> IO a
     -> IO a
 eventuallyUsingDelay delay io = do
-    winner <- race (threadDelay $ 600 * oneSecond) trial
+    winner <- race (threadDelay $ 180 * oneSecond) trial
     case winner of
         Left _ -> fail
-            "waited more than 5min for action to eventually resolve."
+            "waited more than 3min for action to eventually resolve."
```
Comment (KtorZ) [inline lib/core-integration/src/Test/Integration/Framework/DSL.hs:881]: :+1:

**Review by KtorZ — APPROVED** [review]

**Pull request merged**